Unity-Technologies/ScriptableRenderPipeline
# Move TextureCombiner class to proper namespace and add doc.

---
### Purpose of this PR
Documentation was missing for public API of TextureCombiner.
Also, the class was not in a namespace.

---
### Testing status

**Manual Tests**: What did you do?
- Used documentation validator to check that all the public member have valid doc.
- Run the upgrader on some materials to check that it is still working.

No further test in mind, but a review from somebody more used to write api doc would be nice.
